### PR TITLE
create a new standard load balancer

### PR DESCRIPTION
**Pull request by mumian**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 101-load-balancer-standard-create/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+**This template provides an example for how to deploy a standard load balancer.**
+
+The template creates and configures the following Azure resources:
+
+- a load balancer
+- a network security group
+- a virtual network
+- a virtual network interface
+- three virtual machines with each vm in a redundant zone
```
Comment (KumudD) [inline 101-load-balancer-standard-create/README.md]: Please change to "three virtual machines with each vm in a different zone"

File: 101-load-balancer-standard-create/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-load-balancer-standard-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+**This template provides an example for how to deploy a standard load balancer.**
+
+The template creates and configures the following Azure resources:
+
+- a load balancer
```
Comment (KumudD) [inline 101-load-balancer-standard-create/README.md]: Please change to "a Standard Load Balancer"

File: 101-load-balancer-standard-create/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- a network security group
+- a virtual network
+- a virtual network interface
```
Comment (KumudD) [inline 101-load-balancer-standard-create/README.md]: Please replace with - three NICs associated with the backend pool of the load balancer

File: 101-load-balancer-standard-create/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+      "location": "[parameters('location')]",
+      "zones": [
+        "[copyIndex(1)]"
+      ],
+      "copy": {
+        "name": "vmCopy",
+        "count": 3
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Network/networkInterfaces/', concat(parameters('projectName'),'-vm', copyIndex(1), '-networkInterface'))]"
```
Comment (bmoore-msft) [inline 101-load-balancer-standard-create/azuredeploy.json]: use "networkInterfaceCopy"

File: 101-load-balancer-standard-create/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+      "type": "Microsoft.Network/networkInterfaces",
+      "apiVersion": "2018-12-01",
+      "name": "[concat(parameters('projectName'),'-vm', copyIndex(1), '-networkInterface')]",
+      "location": "[parameters('location')]",
+      "copy": {
+        "name": "networkInterfaceCopy",
+        "count": 3
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Network/virtualNetworks/', variables('vNetName'))]",
```
Comment (bmoore-msft) [inline 101-load-balancer-standard-create/azuredeploy.json]: remove concat and use resourceId and the loop name

File: 101-load-balancer-standard-create/azuredeploy.json [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+              "privateIPAllocationMethod": "Dynamic",
+              "subnet": {
+                "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vNetName'), variables('vNetSubnetName'))]"
+              },
+              "publicIpAddress": {
+                "id": "[resourceId(resourceGroup().name, 'Microsoft.Network/publicIpAddresses', concat(parameters('projectName'),'-vm', copyIndex(1), '-publicIp'))]"
+              },
+              "loadBalancerBackendAddressPools": [
+                {
+                  "id": "[concat(resourceId('Microsoft.Network/loadBalancers', variables('lbName')), '/backendAddressPools/', variables('lbBackendPoolName'))]"
```
Comment (bmoore-msft) [inline 101-load-balancer-standard-create/azuredeploy.json]: remove concat() and use resourceId()

File: 101-load-balancer-standard-create/azuredeploy.json [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+                }
+              ]
+            }
+          }
+        ],
+        "networkSecurityGroup": {
+          "id": "[resourceId(resourceGroup().name, 'Microsoft.Network/networkSecurityGroups', variables('nsgName'))]"
```
Comment (bmoore-msft) [inline 101-load-balancer-standard-create/azuredeploy.json]: remove resourceGroup().name

File: 101-load-balancer-standard-create/azuredeploy.json [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "apiVersion": "2018-10-01",
+      "name": "[concat(parameters('projectName'),'-vm', copyIndex(1), '/', 'InstallWebServer')]",
+      "location": "[parameters('location')]",
+      "copy": {
+        "name": "extensionCopy",
+        "count": 3
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Compute/virtualMachines/', concat(parameters('projectName'),'-vm', copyIndex(1)))]"
```
Comment (bmoore-msft) [inline 101-load-balancer-standard-create/azuredeploy.json]: use the loop name

File: 101-load-balancer-standard-create/azuredeploy.json [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+        ],
+        "loadBalancingRules": [
+          {
+            "name": "HTTPLBRule",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id": "[concat(resourceId('Microsoft.Network/loadBalancers', variables('lbName')), '/frontendIPConfigurations/', variables('lbFrontEndName'))]"
+              },
+              "backendAddressPool": {
+                "id": "[concat(resourceId('Microsoft.Network/loadBalancers', variables('lbName')), '/backendAddressPools/', variables('lbBackendPoolName'))]"
```
[the remark this answers is not in the document]
Comment (mumian) [reply 101-load-balancer-standard-create/azuredeploy.json]: Done.

**Comment by mumian** [comment]
@bmoore-msft - hi Brian,  can you take a look of this PR.  Your comments have been incorporated.  Thanks.

**Comment by mumian** [comment]
@bmoore-msft - can you take a look of this PR when you get a chance?

**Pull request merged**